packager/packager.cc

```diff
@@ -598,6 +598,39 @@ class PackageInfoImpl final : public core::packages::PackageInfo {
                 return core::packages::StrictDependenciesLevel::Dag;
         }
     }
+
+    bool importsTransitively(const core::GlobalState &gs, const PackageInfo &otherPkg) const {
+        Timer timeit(gs.tracer(), "importsTransitively");
```

Comment:
Heads up that this timer will get called once per call to `importsTransitively`. If there's no single call to `importsTransitively` that takes longer than like 4 (or maybe 8? I don't remember) milliseconds, then these timers won't even show up (but they will take time to create themselves).

My understanding is that this would be fast enough to dip under that. If we're worried about somehow regressing performance in some portion, it might be worth having a single timer for the whole "strictness + layering" errors section in VisibilityChecker?